eclipsePlugin/test/de/tobject/findbugs/reporter/ThrottledProgressMonitorTest.java

```diff
@@ -0,0 +1,52 @@
+package de.tobject.findbugs.reporter;
```

Comment:
I see, but I still want to put this test at here because:

* According to https://github.com/spotbugs/spotbugs/issues/153#issuecomment-296539032, currently we have no CLI entry to run JUnit test for Eclipse plugin
* It is good habit to run test continuously. It would be nice to add a CLI entry (`./gradlew test` in this case) to test plugin.
* Then we can use Travis CI (or any other CI solution) to keep testing.

I think we can use `eclipsePlugin-test` for scenario test, and this `eclipsePlugin/test` for unit test.
How do you feel?